### DISCUSSION TITLE
Monorepo / subfolders

### DISCUSSION THREAD

**Discussion by jjjuulliiaann** [nuxtrdev/nuxtr-vscode#10] [Ideas]
Hey folks, I would love to use the extension in monorepos where the nuxt installation is inside a subfolder. Would it be a good idea to make that possible or is it maybe already possible somehow? Thanks!

**Comment by adhamfarrag** [comment]
Hey @jjjuulliiaann, already fixed and should be available for next release. Pretty much it will follow the location `srcDir` inside `nuxt.config.ts/js`.

If you have another context, please share it with us and we're happy to consider it.

**Reply by jjjuulliiaann** [reply]
Nice! But it sounds like the `nuxt.config.ts/js` file would still need to be placed in the root folder so that `srcDir` is considered, right? In my use case the whole nuxt install is inside a subfolder including `nuxt.config.ts/js`. An example for that could be a monorepo with separate subfolders for the cms (in `/cms`) and the nuxt frontend (in `/web`).

**Reply by adhamfarrag** [reply]
Ok, different context. We will offer it via a project based setting then. Should be available with next release.

**Reply by jjjuulliiaann** [reply]
Sounds awesome, thank you!

**Comment by adhamfarrag** [comment]
Hello @jjjuulliiaann

`srcDir` and monoRepo are now supported. More on them here https://github.com/nuxtrdev/nuxtr-vscode#srcdir-and-monorepo-support.

Let us now if you face any issues and hhanks for suggesting this feature :)

**Comment by jjjuulliiaann** [comment]
Nice, thanks a lot! Works great so far! Just noticed a small typo in the settings name `nuxtr.monorepoMode.DiretoryName` instead of `...DirectoryName`

**Reply by adhamfarrag** [reply]
Nice catch. Fixed and will be pushed soon. Thanks

**Comment by angelhdzmultimedia** [comment]
### Solved
I thought I had a workspace just because I had a `.vscode/.code-workspace` LOL! I had to save the Untitled workspace as a real workspace first. Now working beautifully thanks!
![image](https://github.com/nuxtrdev/nuxtr-vscode/assets/10191085/ecc3eaea-c141-44b0-82f5-d5a2de51590a)


> Hello!
> 
> How did you get this to work? I tried this by playing with VSCode Nuxt/Nuxtr related settings, but not working for me! https://github.com/nuxtrdev/nuxtr-vscode/discussions/122
>
